docs/advanced-usage/user-quotas/index.md

```diff
@@ -0,0 +1,123 @@
+# User quotas
+
+Since
+[REANA 0.8.0](https://blog.reana.io/posts/2021/release-0.8.0/#cpu-and-disk-quota-accounting),
+it is possible to limit the amount of disk and CPU time available to execute
+workflows, with the help of user quotas.
+
+## Check usage and quota limits
+
+User quotas are optional features of REANA which might not be enabled in your
+cluster. To check if they are enabled and which are the usage limits, you can
+use the `quota-show` command of the `reana-client` command-line tool:
+
+```console
+$ reana-client quota-show --resource disk --report limit
+No limit.
+$ reana-client quota-show --resource disk --report usage
+192512
+$ reana-client quota-show --resource disk
+192512 used
+```
+
+In particular, you can select which resource to consider (`cpu` or `disk`) with
+the `--resource` option and you can choose to show only the usage or the quota
+limit of the given resource with `--report`.
+
+You can also specify the `-h`/`--human-readable` command-line option to display
+the disk or CPU usage in a human-readable format:
+
+```console
+$ reana-client quota-show --resource cpu --report limit -h
+1h
+$ reana-client quota-show --resource cpu --report usage -h
+1m 19s
+$ reana-client quota-show --resource cpu -h
+1m 19s out of 1h used (2%)
+```
+
+The usage of the resources is also displayed on the profile page of the web UI,
+which you can access by clicking on the person-shaped icon in the top-right
+corner of the web interface:
+
+!["Your quota" section of the profile page](/images/ui-profile-your-quota.png)
+
+If the _"Your quota"_ section is not present, it means that user quotas are not
+enabled in your cluster!
```

Comment:
```suggestion
!!! note
    If the _"Your quota"_ section is not present, it means that user quotas are not enabled in your cluster!
```

Similar to https://docs.reana.io/administration/deployment/deploying-at-scale/. This will give you a nice "Note" box.